third-party/linshare/src/test/java/org/apache/james/linshare/LinshareExtension.java

```diff
@@ -133,24 +133,12 @@ public LinshareConfiguration configurationWithJwtFor(LinshareFixture.Credential
 
     @Override
     public void beforeAll(ExtensionContext extensionContext){
-        TechnicalAccountCreationRequest technicalAccount = new TechnicalAccountCreationRequest(TECHNICAL_ACCOUNT.getUsername(),
-            "technical@linshare.org",
-            "true",
-            TECHNICAL_ACCOUNT.getPassword(),
-            "DELEGATION");
-
-        createTechnicalAccount(technicalAccount);
+        createTechnicalAccount(TechnicalAccountCreationRequest.defaultAccount());
     }
 
     private void createTechnicalAccount(TechnicalAccountCreationRequest technicalAccountDTO) {
-        TechnicalAccountDTO technicalAccount = LinshareAPIForTesting.from(ADMIN_ACCOUNT, linshare).createUserAccount(technicalAccountDTO);
-
-        TechnicalAccountGrantPermissionsRequest technicalAccountPermissions = new TechnicalAccountGrantPermissionsRequest(technicalAccount.getUuid(),
-            technicalAccount.getMail(),
-            technicalAccount.getName(),
-            TECHNICAL_PERMISSIONS);
-
-        LinshareAPIForTesting.from(ADMIN_ACCOUNT, linshare).grantTechnicalAccountPermissions(technicalAccountPermissions);
+        TechnicalAccountDTO technicalAccount = LinshareAPIForTesting.from(ADMIN_ACCOUNT, linshare).createTechnicalAccount(technicalAccountDTO);
+        LinshareAPIForTesting.from(ADMIN_ACCOUNT, linshare).grantTechnicalAccountPermissions(new TechnicalAccountGrantPermissionsRequest(technicalAccount));
```

Comment:
And I guess you can extract the API into a variable:
```java
        LinshareAPIForTesting linshareAPI = LinshareAPIForTesting.from(ADMIN_ACCOUNT, linshare);
        TechnicalAccountDTO technicalAccount = linshareAPI.createTechnicalAccount(technicalAccountDTO);

        linshareAPI.grantTechnicalAccountPermissions(
            new TechnicalAccountGrantPermissionsRequest(technicalAccount));
```

Reply:
oh, right.I will reverse it